tests/js/index.js

```diff
@@ -41,10 +41,11 @@ if (Realm.Sync) {
     TESTS.SessionTests = require('./session-tests');
 }
 
-function node_require(module) { return require(module); }
+function node_require(module) { return require(module); }
 
 // If on node, run the async tests
-if (typeof process === 'object' && process + '' === '[object process]') {
+global.isNodeProcess = typeof process === 'object' && process + '' === '[object process]'
```

Comment:
I think this runs in the browser as well, when running tests with chrome debugging? In which case, I guess `global` is undefined.